dataprofiler/profilers/profile_builder.py

```diff
@@ -1160,7 +1160,9 @@ def __init__(self, data, samples_per_update=None, min_true_samples=0,
         self.row_has_null_count = 0
         self.row_is_null_count = 0
         self.hashed_row_dict = dict()
-        self._profile = dict()
+        self._profile = []
+        self._col_name_to_idx = defaultdict(list)
+        self._duplicate_cols_present = False
```

Comment:
do we still need `_duplicate_cols_present`?

Reply:
Removed